lib/management/models/index.ts

```diff
@@ -0,0 +1,350 @@
+/*
```

Comment:
Do you want to put the generated stuff under `lib/management/generated`. In that way if we want to write custom stuff it will not get mixed with generated stuff.

Reply:
Done.